### ISSUE TITLE
span labels not emitted correctly to rtf

### ISSUE THREAD

**Issue by gmbecker** [rstudio/gt#1054]
To reproduce:
```
library(gt)

mydf <- data.frame(x = 5, y = "hi", z = "what?")
mygt <- mydf |>
  gt() |>
  tab_spanner(label ="spaannning", columns = c(y, z))

rtf_str <- as_rtf(mygt)
```

Then `cat` `rtf_str` to the file of your choice and open the file in an editor/viewer that supports RTF.

For me, I tried TextEdit  (Version 1.17 (380.2)) and LibreOffice 7.3.5.2. Both of these display (different) incorrect tables:

![gt_rtf_loaded_screenshot](https://user-images.githubusercontent.com/908721/189775730-aa2a41f9-58c6-45c2-9794-7be8b66c1b3f.png)

Looking at `rtf_str` we see:

```
 rtf_str
{\rtf\ansi\ansicpg1252{\fonttbl{\f0\froman\fcharset0\fprq0 Courier New;}{\f1\froman\fcharset0\fprq0 Times;}}{\colortbl;\red211\green211\blue211;}

\paperw12240\paperh15840\widowctrl\ftnbj\fet0\sectd\linex0
\lndscpsxn
\margl1440\margr1440\margt1440\margb1440
\headery720\footery720\fs20

\trowd\trrh0\trhdr

\pard\plain\uc0\qc\clvertalc \clpadfl3\clpadl25 \clpadft3\clpadt85 \clpadfb3\clpadb25 \clpadfr3\clpadr85 \cellx3120
\intbl {\f0 {\f0\fs20 }}\cell

\pard\plain\uc0\qc\clvertalc \clpadfl3\clpadl25 \clpadft3\clpadt85 \clpadfb3\clpadb25 \clpadfr3\clpadr85\clbrdrb\brdrs\brdrw20\brdrcf1\clmgf \cellx6240
\intbl {\f0 {\f0\fs20 spaannning}}\cell

\pard\plain\uc0\qc\clvertalc \clpadfl3\clpadl25 \clpadft3\clpadt85 \clpadfb3\clpadb25 \clpadfr3\clpadr85\clbrdrb\brdrs\brdrw20\brdrcf1\clmrg \cellx9360
\intbl {\f0 {\f0\fs20 spaannning}}\cell

\row

<snip>

```
Note the value `spaannning` is in both the `\clmfg` (or merge start) cell ***and*** the following `\clmrg` (merge into preceding cell); I believe this is incorrect and that `\clmrg` cells should be blank. 

If I edit the file so that the second spanning is removed and everything else is identical, both editors load and display the table correctly:
![gt_rtf_edited_screenshot](https://user-images.githubusercontent.com/908721/189776316-9bebaafd-209c-4cd8-a49e-ba05ca9162a6.png)

```
> sessionInfo()
R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] gt_0.7.0

loaded via a namespace (and not attached):
 [1] magrittr_2.0.3   tidyselect_1.1.2 munsell_0.5.0    colorspace_2.0-3
 [5] R6_2.5.1         rlang_1.0.2      fastmap_1.1.0    fansi_1.0.3     
 [9] dplyr_1.0.8      grid_4.1.2       gtable_0.3.0     utf8_1.2.2      
[13] cli_3.3.0        DBI_1.1.2        htmltools_0.5.2  ellipsis_0.3.2  
[17] assertthat_0.2.1 digest_0.6.29    tibble_3.1.6     lifecycle_1.0.1 
[21] crayon_1.5.1     purrr_0.3.4      ggplot2_3.3.5    vctrs_0.4.0     
[25] glue_1.6.2       compiler_4.1.2   pillar_1.7.0     generics_0.1.2  
[29] scales_1.2.1     pkgconfig_2.0.3 
```

**Comment by rich-iannone** [comment]
Thanks Gabe, I will fix this soon!

**Comment by gmbecker** [comment]
@rich-iannone should I take from the above that the fix for this is unlikely to be available soon? I need to know whether I should wait on `gt` or scrap it as an avenue-to-rtf for `rtables` tables. cc @waddella

**Comment by rich-iannone** [comment]
This will be the next thing to be fixed. That milestone change was wrong (now corrected).

**Comment by gmbecker** [comment]
Thanks @rich-iannone will test soon
